docs-chef-io/content/server/upgrade_server_ha_v2.md

```diff
@@ -165,11 +165,110 @@ Upgrading from Chef Backend 1.x to Chef Backend 2.x requires full cluster downti
 
 | Running Version | Upgrade To Version | Upgrade Type |
 |---------|---------|------|
-| 2.x | 2.x | Rolling |
-| 2.1 | 3.0 | Rolling |
+| 2.0 | 2.1 | Rolling |
+| 2.1 | 3.0 | Rolling if the created string in curl -XGET localhost:9200/chef/_settings/index.version.created*?pretty\&human is 5.6.16 or greater|
+| 2.1 | 3.0 | Downtime if the created string in curl -XGET localhost:9200/chef/_settings/index.version.created*?pretty\&human is 5.4.1 or lesser|
 | 2.0 | 3.0 | Downtime |
```

Comment:
This downtime implies it is still a rolling type of upgrade for the backend, but the frontends need to be down.